docs/guide/acquiring_data/fido.rst

```diff
@@ -46,7 +46,6 @@ variable set to the Fido search, in this case, result::
     <BLANKLINE>
     3 Results from the LYRAClient:
          Start Time           End Time      Source Instrument Wavelength
-           str19               str19         str6     str4       str3
```

Comment:
Can the change that gets rid of these bits be put in it's own PR?

Reply:
Any reason?

Reply:
This change is tied up with standardising the `__str__`` and `__repr__` methods in `BaseQueryResponse`.

As for large diff, can I suggest you try and review the PR commit by commit if you want to have some logical separation between changes. I will add a changelog(s) to each commit in a bit to hopefully make it clearer as well.

Reply:
It looks like at least commits 2 and 3 could easily be simple individual PRs - I'd be happy to cherry-pick them out myself. Definitely appreciate that everything has to be here to start with to get the end product working, but I don't have the mental capacity to properly review a 500 line PR.